src/index.ts

```diff
@@ -482,10 +491,11 @@ export class ServerlessClamscan extends Construct {
         FnName: download_defs.functionName,
       },
     });
-
+    
     if (props.buckets) {
       props.buckets.forEach((bucket) => {
-        this.addSourceBucket(bucket);
+        const bucketAdditionalInfo = props.bucketsEventNotificationProps?.find(x => x.bucketName === bucket.bucketName);
+        this.addSourceBucket(bucket, bucketAdditionalInfo);
```

Comment:
**suggestion:** again, we could use _options_ for naming, which would be more consistent
```suggestion
        const bucketEventNotificationOptions = props.bucketsEventNotificationProps?.find(x => x.bucketName === bucket.bucketName);
        this.addSourceBucket(bucket, bucketEventNotificationOptions);
```